src/helpers/normalizeHomeDomainUrl.ts

```diff
@@ -0,0 +1,16 @@
+export const normalizeHomeDomainUrl = (homeDomain: string): URL => {
+  let _homeDomain = homeDomain;
+
+  // default localhost to http instead of https
+  if (_homeDomain.includes("localhost")) {
+    _homeDomain = _homeDomain.startsWith("http")
+      ? _homeDomain
+      : `http://${_homeDomain}`;
+  } else {
+    _homeDomain = _homeDomain.startsWith("http")
+      ? _homeDomain
+      : `https://${_homeDomain}`;
+  }
```

Comment:
I spent a few min coming up with a way to do this in fewer lines, but I honestly think keeping it the way you wrote it is more clear haha

Reply:
This was the shorter version I came up with, which is actually kind of confusing:
```  
if (!_homeDomain.startsWith("http")) {
    _homeDomain = _homeDomain.includes("localhost") ? `http://${_homeDomain}` : `https://${_homeDomain}` 
  }
```

Reply:
It's funny, I did exactly the same thing and went with the more clear solution. With simple `if else` there is no question about what is going on.
There are two things ( `localhost` and `.startsWith("http")`) we need to check because provided home domain could have `http(s)` in which case we leave it as is.